repo2docker/buildpacks/julia/semver.py

```diff
@@ -0,0 +1,162 @@
+"""
```

Comment:
Just curious, why did you rename this from `julia_semver` to `semver`? I _think_ the logic in this file is fairly specific to `julia`, and it'd be a shame if someone accidentally used it for parsing semvers in an unrelated context.. What do you think?

Reply:
Also, we're importing a package named `semver` from a file named `semver` which just feels weird to me. But I don't write much python these days, so I might also just not be as comfortable in python land.

Reply:
Mostly because I don't think anyone will use it outside of this part of the package. And if they do they'd be writing something like `from repo2docker.buildpacks.julia.semver import foo` or `from repo2docker.buildpacks.julia import semver`. If you want to use this alongside the other semver package you can always do `from repo2docker.buildpacks.julia import semver as julia_semver`.

Relying on the namespacing of python modules.